test/torchaudio_unittest/rnnt/autograd_impl.py

```diff
@@ -0,0 +1,69 @@
+from typing import Callable, Tuple
+import torch
+from torch import Tensor
+from torch.autograd import gradcheck
+from torchaudio_unittest.common_utils import (
+    TestBaseMixin,
+)
+from .utils import get_numpy_random_data, numpy_to_torch
+from torchaudio.prototype.rnnt_loss import RNNTLoss, rnnt_loss
+
+
+class Autograd(TestBaseMixin):
+    def assert_grad(
+            self,
+            transform: Callable[..., Tensor],
+            inputs: Tuple[torch.Tensor],
+            *,
+            enable_all_grad: bool = True,
+    ):
+        inputs_ = []
+        for i in inputs:
+            if torch.is_tensor(i):
+                i = i.to(dtype=self.dtype, device=self.device)
+                if enable_all_grad:
+                    i.requires_grad = True
+            inputs_.append(i)
+        assert gradcheck(transform, inputs_, eps=1e-06, atol=1e-05)
+
+    def test_rnnt_loss_small(self):
```

Comment:
let's try small epsilon?

Reply:
cc @albanD

Reply:
When the numerical value goes crazy, it usually is because you try to gradcheck at a point where the function is not continuously differentiable.
So you might want to give different inputs to make sure you're at a point that is.

Reply:
offline discussion from @albanD: since float64 is not implemented, the epsilon needs to be changed from the default to something like 1e-3, and tolerance to ~1e-2

Reply:
eps = 1e-2: P414383132
eps = 1e-3: P414382964